### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by FPolillo**
Zeh, I have edited this page to make it more user friendly. Kindly approve it, and then I will just point users to this page from the help.

### CODE REVIEW

**Review by FPolillo — COMMENTED** [review]
I have updated the article to make it more user friendly.

File: tasks/connectors/edgescan/README.md [hunk 1]
```diff
@@ -15,20 +15,20 @@ This task will communicate with the Edgescan and Kenna APIs. In order to do so i
 - Kenna API key
 - Kenna connector ID
 
-## Running the task
+## Running a Task
 
-More in depth details about running the task are available [here](https://github.com/KennaSecurity/toolkit/blob/master/README.md).
-These are some quick examples:
+Fo more detailed inforamtion about running the task, see [here](https://github.com/KennaSecurity/toolkit/blob/master/README.md).
```
Comment (jgarber-cisco) [inline tasks/connectors/edgescan/README.md:20]: Two typos

File: tasks/connectors/edgescan/README.md [hunk 2]
```diff
@@ -48,10 +48,10 @@ By default the vulnerabilities will be both application and network types. Eithe
 
 ## Data Mappings
 
-Edgescan assets do not map directly to Kennna assets due to Edgescan assets being more flexible in their definition.
+Edgescan assets do not map directly to Kennna assets because Edgescan assets are more flexible in their definition.
 Edgescan location specifiers and hosts are more like Kenna assets. Location specifiers define the location and hosts hold extra information.
 Not all location specifiers have a host, and not all vulnerabilities have a directly related location specifier.
-The connector makes use of the data from all 3 of these sources to create the correct corresponding Kenna assets.
+To create the correct corresponding Kenna assets, the connector makes use of the data from all theree of the following sources:
```
Comment (jgarber-cisco) [inline tasks/connectors/edgescan/README.md:54]: Typo

File: tasks/connectors/edgescan/README.md [hunk 3]
```diff
@@ -114,8 +114,8 @@ The connector makes use of the data from all 3 of these sources to create the co
 | cve_identifiers     | definition.cves                |                                                      |
 | cwe_identifiers     | definition.cwes                |                                                      |
 
-## For devs
+## For Developers
 
-Pass in this env variable to make the task talk to `localhost:3000` instead of `live.edgescan.com`:
+Use this pass env variable to make the task comminicate with `localhost:3000` instead of `live.edgescan.com`:
```
Comment (jgarber-cisco) [inline tasks/connectors/edgescan/README.md:119]: "Use this environment variable..."

**Review by jgarber-cisco — CHANGES_REQUESTED** [review]

**Review by jgarber-cisco — APPROVED** [review]

**Pull request merged**